main.js

```diff
@@ -1494,7 +1504,7 @@ function stop(debug) {
     }
 
     window.onresize = resizeTv;
-    resizeTv();
+    window.setTimeout(resizeTv, 500);
```

Comment:
Is this the hopeful fix for the "my tv is too small?" Maybe we can reduce this number, else there's a half second with the wrong-sized TV, right? Either that or have both a `, 1)` AND a `, 500)`